README.md

```diff
@@ -91,6 +91,102 @@ Provisioner](https://www.packer.io/docs/provisioners/shell.html) runs the
 necessary configuration tasks.  Then, Packer creates an AMI from the instance
 and terminates the instance after the AMI is created.
 
+### Container Image Caching
+
+Optionally, some container images can be cached during the AMI build process in order to reduce the latency of the node getting to a `Ready` state when launched. 
+
+To turn on container image caching:
+
+```
+cache_container_images=true make 1.23
+```
+
+When container image caching is enabled, the following images are cached:
+ - 602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/eks/kube-proxy:<K8s-Version>-eksbuild.1
+ - 602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/eks/kube-proxy:<K8s-Version>-minimal-eksbuild.1
+ - 602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/eks/pause:3.5
+ - 602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/amazon-k8s-cni-init:<default and latest>
+ - 602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/amazon-k8s-cni:<latest and latest>
+
+Since the VPC CNI is not versioned with K8s itself, the latest version of the VPC CNI and the default version, based on the response from the EKS DescribeAddonVersions at the time of the AMI build, will be cached. 
+
+The images listed above are also tagged with each region in the partition the AMI is built in, since images are often built in one region and copied to others within the same partition. Images that are available to pull from an ECR FIPS endpoint are also tagged as such (i.e. `602401143452.dkr.ecr-fips.us-east-1.amazonaws.com/eks/pause:3.5`).
```

Comment:
To avoid confusion, maybe we should put reference link to https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html to let reader aware of that `${AWS_ACCOUNT_ID}` for registry. Since it is not all referenced to `602401143452 `.

WDYT?